### PR TITLE
[OSCD] Added a rule to detect persistence using registry keys

### PR DESCRIPTION
**Pull request by V3T0**
Added a rule to detect persistence by using the registry key: HKLM\SOFTWARE\Microsoft\Active Setup\Installed Components. An attacker can add payload in StubPath value and execute it via runonce.exe

### CODE REVIEW

**Comment by yugoslavskiy** [comment]
Hello @V3T0! 
Thank you for your contribution (: 

Could you rebase your PR so it would have only your commit? Here is an [instruction](https://stackoverflow.com/questions/34027850/how-to-pull-request-a-specific-commit).

File: rules/windows/registry_event/sysmon_runonce_persistence.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+date: 2020/10/15
+references:
+    - https://twitter.com/pabraeken/status/990717080805789697
+    - https://github.com/LOLBAS-Project/LOLBAS/blob/master/yml/OSBinaries/Runonce.yml
+tags:
+    - attack.defense_evasion
+    - attack.t1112
+logsource:
+    product: windows
+    service: sysmon
```
Comment (yugoslavskiy) [inline rules/windows/registry_event/sysmon_runonce_persistence.yml:15]: I believe you can also change it to the `registry_event ` here (:

File: rules/windows/registry_event/sysmon_runonce_persistence.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+title: Run Once task configuration in the registry
+id: c74d7efc-8826-45d9-b8bb-f04fac9e4eff
+description: Rule to detect the configuration of Run Once registry key. Configured payload can be run by runonce.exe /AlternateShellStartup
+author: Avneet Singh '@v3t0_'
```
Comment (yugoslavskiy) [inline rules/windows/registry_event/sysmon_runonce_persistence.yml:4]: you need to quote the entire `author` field, this way:

```
author: 'Avneet Singh'@v3t0_ oscd.community'
```

File: rules/windows/registry_event/sysmon_runonce_persistence.yml (hunk 1 above)
Comment (V3T0) [reply rules/windows/registry_event/sysmon_runonce_persistence.yml:15]: Thank you for the review. :) 
I've changed it as per your suggestions. I can't rebase as per the instructions. It says that **The previous cherry-pick is now empty, possibly due to conflict resolution.**
If I do a git commit to the same branch, would it be okay?

Comment (V3T0) [reply rules/windows/registry_event/sysmon_runonce_persistence.yml:15]: @yugoslavskiy

File: rules/windows/registry_event/sysmon_runonce_persistence.yml (hunk 2 above)
Comment (V3T0) [reply rules/windows/registry_event/sysmon_runonce_persistence.yml:4]: I've changed it and added the oscd.community too :)

**Comment by V3T0** [comment]
closing this PR and will issue a new PR

**Pull request closed without merging**